content/release/10.1.4.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: FRRouting Release 10.1.4
+Release: 10.1.4
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-10.1.4
+Date: 2025-08-02
+---
+
+FRRouting Release 10.1.4 Available for Download
+===============================================
+
+Debian Packages - https://deb.frrouting.org
+
+RPM Packages - https://rpm.frrouting.org
+
+Snaps - https://snapcraft.io/frr
+
+Docker - [quay.io/frrouting/frr](https://quay.io/repository/frrouting/frr/manifest/sha256:79020316ce1fcd26be1393d46a6fbf49c0994fd29dce556829b474ae3e20b8b9)
+
+## What's Changed
+
+* ospf6d: Disable and delete OSPFv3 areas that no longer have interfaces or configuration. (backport #18393) by @mergify[bot] in https://github.com/FRRouting/frr/pull/18424
```

Comment:
can we drop  all mergify stuff? ` by @mergify[bot] in https://github.com/FRRouting/frr/pull/18424`